### PR TITLE
Export claims XML for parallel testing

### PR DESCRIPTION
**Pull request by dfaulken**
Requested by @sherson.

Allows us to parallel test the XML approach while keeping the existing integration intact.

Definitely open to suggestions on more sophisticated methods.

### CODE REVIEW

**Comment by dfaulken** [comment]
Addresses #180.

File: app/models/incident.rb [hunk 1]
```diff
@@ -102,9 +104,55 @@ def csv_row
     row
   end
 
+  def export_claims_xml!
+    report = driver_incident_report
+    builder = Nokogiri::XML::Builder.new do |doc|
+      doc.claims_incident_data do
+        doc.incident do
+          doc.date_entered        Date.today.strftime('%Y-%m-%d')
```
Comment (fermion) [inline app/models/incident.rb]:
```suggestion
          doc.date_entered        created_at.strftime('%Y-%m-%d')
```
?

Comment (dfaulken) [reply app/models/incident.rb]: That was my thought originally too, but the specification is that this should be the date that it is entered into the external system (i.e. whenever we're sending the record along).

**Comment by dfaulken** [comment]
Thanks @fermion and @Anbranin !

**Pull request merged**